db/migrate/20150624004126_add_invocation.rb

```diff
@@ -0,0 +1,49 @@
+class AddInvocation< ActiveRecord::Migration
+  def change
+    create_table :targetings do |t|
+      t.string :search_query
+      t.references :bookmark
+      t.references :user
+      t.timestamps
+    end
+
+    add_index :targetings, [:bookmark_id, :user_id], :name => "targetings_bookmark_user_ids"
```

Comment:
Shouldn't this be two separate indexes, since we have the foreign keys on them separately?